src/main/java/org/ibp/api/java/ruleengine/RulesPostProcessor.java

```diff
@@ -17,8 +19,8 @@
  */
 @Component
 @ComponentScan(basePackages = {
-	"org.generationcp.commons.ruleengine.naming.expression",
-	"org.generationcp.commons.ruleengine.naming.rules",
+	"org.generationcp.commons.ruleengine.coding.expression",
```

Comment:
Why did we remove component scanning in org.generationcp.commons.ruleengine.naming.expression and org.generationcp.commons.ruleengine.naming.rules?

Reply:
I removed them because they are not used in BMSAPI and to avoid conflict with coding expressions.